source/fundamentals/crud/read-operations/query-document.txt

```diff
@@ -444,7 +442,7 @@ Additional Information
 ----------------------
 
 .. For information on specifying a geospatial query, see the guide on
-.. :doc:`Geospatial Data </fundamentals/crud/read-operations/geospatial>`.
+.. :ref:`golang-geo`.
```

Comment:
Suggestion: This can be uncommented now that the geo page is live on all versions!

Reply:
fixed! back to you!